backend/index.js

```diff
@@ -27,7 +27,7 @@ mongoose.connection.on('error', dbConnectionFailure);
 /**
  * Error handler
  */
-const errorHandler = (err, req, res, next) => {
+const errorHandler = (err, req, res) => {
```

Comment:
Why did we remove next here?

Reply:
not sure, I think it was not being used anywhere and was giving an error, have added it again